docs/self-hosted/kubernetes-maintenance.md

```diff
@@ -31,3 +31,10 @@ kubectl kots admin-console upgrade -n NAMESPACE
 
 Replace `NAMESPACE` with the namespace in your cluster where KOTS is installed.
 
+### Updating or Replacing the SSL Certificate
```

Comment:
```suggestion
### Update or replace the SSL certificate
```

Reply:
I took this title from `self-hosted/maintenance.md`, but you are right, this seems off. I will fix it in both places.

Reply:
thanks!